lib/locale/pl_PL/strings.php

```diff
@@ -2600,6 +2600,9 @@
 $_LANG['VLAN Description'] = 'Opis VLAN-u';
 $_LANG['VLAN ID'] = 'Identyfikator VLAN-u';
 $_LANG['Add VLAN'] = 'Dodaj VLAN';
+$_LANG['Assigned Network node'] = 'Przypisany węzeł sieciowy';
+$_LANG['Network Node'] = 'Węzeł sieciowy';
+$_LANG['Network Node ID'] = 'Nr węzła sieciowego';
```

Comment:
Nr? Źle brzmi. To na pewno nie jest numer.

Reply:
Może być po polsku również `ID węzła sieciowego`. To nie będzie nic złego.

Reply:
+1

Reply:
Poprawione.